doc/mysql-handler.md

```diff
@@ -41,15 +41,15 @@ CREATE TABLE IF NOT EXISTS `users` (
 
 ```
 
--- Create administration user
+-- Create administration user {username : admin , password : admin}
```

Comment:
Change the password field to hash mode, When evaluation validation login for user based on hash field.